### PR TITLE
Expose useful table functions

### PR DESCRIPTION
**Pull request by dxparker**
addColumn/addRow needed to change table without manipulating selection.
inSameTable needed to enable/disable table features on selection change.

### CODE REVIEW

**Comment by eshvedai** [comment]
I've exported some of the util functions, including `inSameTable`, but I'm hesitant about exporting those commands as they don't have the same `fn(state, dispatch)` API as the rest of PM commands. What's your opinion on that, @marijnh ?

**Comment by marijnh** [comment]
Depends on how clean/small you want to keep the interface. Exporting non-command functions is not, in principle, a problem.

**Comment by dxparker** [comment]
@eshvedai: good point that addColumn and addRow do not match the command interface. These functions could be moved to the util file and exported from there to maintain consistency. Would you like me to add that change to this PR?

File: src/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import {Plugin} from "prosemirror-state"
 
 import {handleTripleClick, handleKeyDown, handlePaste, handleMouseDown} from "./input"
-import {key} from "./util"
+import {key, cellAround, inSameTable} from "./util"
```
Comment (eshvedai) [inline src/index.js]: I think this change is not needed anymore, look at https://github.com/ProseMirror/prosemirror-tables/pull/40/files#diff-1fdf421c05c1140f6d71444ea2b27638L69

**Comment by eshvedai** [comment]
@dxparker I think if you need `addColumn`, `addRow`, you might also want to export `selectedRect`, right? 

Just wondering why `addRowBefore`/`addRowAfter` is not enough for your use case? What do you mean by manipulating selection exactly?

File: src/index.js (hunk 1 above)
Comment (dxparker) [reply src/index.js]: Completely agree, thanks for pointing this out. These two imports have been removed from the PR.

**Comment by dxparker** [comment]
@eshvedai the `addRowBefore`/`addRowAfter` commands insert rows based on the position of the cursor in the state argument.

I'd like to use `addRow` from UI components (imagine a "+" icon) hovering around/over the beginning of each row to insert a new row when an icon is clicked regardless of which row the users selection is currently in and without having to create a state that places the cursor in an adjacent row followed by a state that restores the selection to whatever it was before the click.

**Comment by dxparker** [comment]
Hi @eshvedai, is this PR still under consideration?

**Comment by eshvedai** [comment]
sorry didn't have time to get back to your PR, merged!

**Pull request merged**